packages/types/src/metrics.ts

```diff
@@ -5,27 +5,21 @@ import type { Primitive } from './misc';
  * An abstract definition of the minimum required API
  * for a metric instance.
  */
-export abstract class MetricInstance {
+export interface MetricInstance {
```

Comment:
Because it was incorrect. We do not want this to be an abstract class. Nothing actually extended it. I made this change specifically in this PR because previously the `value` parameter in the `add` method was unused, causing linting errors.